engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -150,35 +152,21 @@ case class WorkflowExecutionActor(workflowDescriptor: EngineWorkflowDescriptor,
     FSM.NullFunction
   }
   when(WorkflowExecutionFailedState) {
-    alreadyFailedMopUp
```

Comment:
Once in a while I see a name that makes me chuckle and feel the need to go look up who created it. 99.99999999999999% of the time the answer is @cjllanwarne